foundry-bin/update.sh

```diff
@@ -43,7 +60,7 @@ get_hash() {
 
 cat > releases.nix << EOF
 {
-  version = "0.0.0";
+  version = "0.0.0-${SCHEDULE}";
```

Comment:
Not sure this is a great idea, haven't had a chance to do a full build yet.

Reply:
Yeah ideally this version would match the release version, but it would probably involve parsing some toml from the source (maybe getting it through the `raw.githubusercontent.com` URL). Anyway not trivial